packages/LUISGen/src/Program.cs

```diff
@@ -67,53 +68,58 @@ static void Main(string[] args)
+                            case "-ts":
+                                {
+                                    ++i;
+                                    var name = NextArg(ref i, args, optional: true);
+                                    if (name != null)
                                     {
-                                        ++i;
-                                        var name = NextArg(ref i, args, optional: true);
-                                        if (name != null)
+                                        var lastDot = arg == "-cs" ? name.LastIndexOf('.') : -1;
+                                        if (lastDot == -1)
                                         {
-                                            var lastDot = arg == "-cs" ? name.LastIndexOf('.') : -1;
-                                            if (lastDot == -1)
-                                            {
-                                                className = name;
-                                            }
-                                            else
-                                            {
-                                                space = name.Substring(0, lastDot);
-                                                className = name.Substring(lastDot + 1);
-                                            }
+                                            className = name;
+                                        }
+                                        else
+                                        {
+                                            space = name.Substring(0, lastDot);
+                                            className = name.Substring(lastDot + 1);
                                         }
-                                        outType = arg.Substring(1);
-                                    }
-                                    break;
-                                case "-o":
-                                    {
-                                        ++i;
-                                        outPath = NextArg(ref i, args);
                                     }
-                                    break;
-                                default:
-                                    Usage();
-                                    break;
-                            }
+                                    outType = arg.Substring(1);
+                                }
+                                break;
+                            case "-o":
+                                {
+                                    ++i;
+                                    outPath = NextArg(ref i, args);
+                                }
+                                break;
+                            case "-v":
+                            case "--version":
+                                Console.WriteLine($"LUISGen version {System.Reflection.Assembly.GetExecutingAssembly().GetName().Version}");
```

Comment:
Is this the right value? 

Most people will have installed LuisGen via an NPM package that'll be versioned like "1.2.3". The assembly version is a quad, and will likley have a different number than the NPM package. 

Is there a way to get the package from here?